### PR TITLE
style(wash-cli): main help is colorized

### PR DESCRIPTION
**Pull request by markkovari**
## Feature or Problem
Using colours for the main help. 🎉 

although I have a question: compared to the "not overwritten" version of the help this command is not in the list of commands when using the custom 🤔 

I know it i behind experimental flag, but was it intentionally hidden from the cli?

 > capture      Capture and debug cluster invocations and state
 
 
and I updated the comments above the (Sub)Commands to reflect the plain text version. 
Some of them have been out of sync (beside the `NOTE`).

Did a minimal research too if there is a dynamic stuff or not, but not really 😞 manual maintenance it is.
 
## Related Issues

closes: #2873 

### Manual Verification

```console
cd crates/wash-cli
```
```console
cargo run --
```

### CODE REVIEW

**Comment by vados-cosmonic** [comment]
Hey @markkovari this looks great -- I think you're missing the `color_print` dep addition though :)

That said, we already have [`termcolor`](https://crates.io/crates/termcolor) so it might be nice to use that instead of introducing a new dep

**Comment by markkovari** [comment]
Thanks @vados-cosmonic, yeah it was pretty late 😄 I used the mentioned dependencies and added a dynamic padding to avoid width typos for the commands. 

Although there must be a better way to pad the strings, so whoever knows by heart please help me out.

File: crates/wash-cli/src/bin/wash.rs [hunk 1]
```diff
@@ -43,7 +46,54 @@ use wash_cli::style::WASH_CLI_STYLE;
...
+impl Display for HelpTopic {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        const PADDING_AFTER_LONGEST_SPACES: usize = 3;
+        writeln!(f, "{}", self.name.green().bold())?;
+        let longest_command_length = self
+            .commands
+            .iter()
+            .map(|(name, _)| name.len())
+            .max()
+            .unwrap_or(25)
```
Comment (vados-cosmonic) [inline crates/wash-cli/src/bin/wash.rs:64]: you could pull this out as a constant as well!

**Review by vados-cosmonic — APPROVED** [review]
This looks great for a start @markkovari thanks!

I think the hard-coded spacing isn't the end of the world -- it's certainly better than doing it completely manually to start.

**Review by brooksmtownsend — APPROVED** [review]

**Comment by markkovari** [comment]
I might need some help here 😶‍🌫️ one test failed and removed this item from the queue. Although should have no effect.

🥺 @vados-cosmonic or @brooksmtownsend this might need a non-violent re-kick
Thank you

**Pull request merged**